mep/accounts/migrations/0025_subevent_item_to_generic.py

```diff
@@ -0,0 +1,67 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.20 on 2019-02-22 21:01
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+def copy_items_to_generic_event(apps, schema_editor):
+    '''
+    Copy items from event subclasses that have them to generic
+    event item.
+    '''
```

Comment:
This looks simple enough that I think a full-on migration test might be unnecessary. I do think it might be worth refactoring slightly to do a count of purchases and borrows with items, and then the same withgeneric_items, and `assert`ing that they're the same such that the migration will fail out if they aren't as a warning to us in test/QA.

Code-wise, I can't see how that would possibly happen, but it's reassuring to know numbers are the same. I